src/System.IO.FileSystem/src/System/IO/UnixFileStream.cs

```diff
@@ -418,10 +418,7 @@ protected override void Dispose(bool disposing)
                         // Since we still have the file open, this will end up deleting
                         // it (assuming we're the only link to it) once it's closed, but the
                         // name will be removed immediatly.
```

Comment:
:bulb: if spell checker extension is installed, https://visualstudiogallery.msdn.microsoft.com/7c8341f1-ebac-40c8-92c2-476db8d523ce, it would probably highlight all the typos. Then the committer can fix typos for only the touched files in the commit / change-set. Soon enough, there won't be any typo left in the entire source tree.